src/Login/Login.jsx

```diff
@@ -157,7 +157,7 @@ class Login extends React.Component {
                   // over the login options' names (e.g. "The University of
                   // Chicago") and not the actual option values, which are
                   // URLs.
-                  loginOptions[i].length > 1 && (
+                  ((dropdownSingleLogin && p.name === "InCommon Login") || loginOptions[i].length > 1) && (
```

Comment:
nitpick: the config option name `dropdownSingleLogin` sounds like it should have effects on all login dropdowns instead of just incommon login

Reply:
It doesn't make much sense in the context of any other login options...

And this make me think that I can make it a little bit more generic.

Reply:
@mfshao @jarvisraymond-uchicago I think new version is much cleaner and generic. :)